screens/rating.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+import { Font } from 'expo';
+import FontLoad from './components/FontLoad';
+
+const window = Dimensions.get("window")
+
+export default class Rating extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  componentWillMount = () => {
+    FontLoad.then((res) => {
+      this.setState({ fontLoaded: true });
+    })
+  }
+
+  stars = () => {
+    let stars = [1, 2, 3, 4, 5]
+    let starMap = stars.map((item) => {
+      return (
+        <View key={item}>
+          <Image style={styles.starIcon}
+              source={require('./images/star.png')} />
+        </View>
+      )
+    })
+    return starMap;
+  }
+
+  render() {
+    return (
+      <View>
+      {
+        this.state.fontLoaded ? (
+          <View style={styles.container}>
+            <View style={styles.topBanner} />
+            <Image style={styles.shoppingCarticon}
+                source={require('./images/shopping-cart.png')} />
+            <Image style={styles.personIcon}
+                source={require('./images/profile_person.png')} />
+            <Text style={styles.thankYouforText}>Thank U for</Text>
+            <Text style={styles.usingText}>using</Text>
+            <Text style={styles.shopuText}>shopU!</Text>
+            <View style={styles.ratingBox} />
+            <Text style={styles.delivererText}>Deliverer:</Text>
+            <View style={{width: 300, flexDirection: 'row', marginTop: 800, marginLeft: 185}}>
```

Comment:
You shouldn't have inline styles - make a new style below

Also use flexboxes - you shouldn't be using `800px` margins to position items

Reply:
Should I make it come down like 3/4 of the page (something along those lines)?

Reply:
Like a percentage instead of pixels

Reply:
I would first use flexboxes first and then use margins to tweak the small details